classes/Provider/YamlConfigDriver.php

```diff
@@ -15,4 +14,9 @@ public function load($filename)
         $config = Yaml::parse(file_get_contents($filename));
         return $config ?: [];
     }
+
+    public function supports($filename)
+    {
+        return (bool) preg_match('#\.ya?ml(\.dist)?$#', $filename);
```

Comment:
This might not be an issue, but maybe you'd want to confirm that the file actually has a name, rather than allowing a file named ".yaml"/".yaml.dist"? I suppose it could be argued that those are both perfectly cromulent filenames.